### PR TITLE
adding collapsable table row modifier

### PR DESCRIPTION
**Pull request by williamdingwall**

### CODE REVIEW

File: src/components/Panel.scss [hunk 1]
```diff
@@ -240,3 +241,69 @@ $Panel-header-text-color: $cu-info !default;
...
 }
+
+// * 4. Collapsable Panel Rows
+
+// 1. The child row makes up the bottom border instead of parent row
+// 2. Style is nested for specificity
+// 3. The Parent row needs to be targeted seperately from a static Panel row
+
+.Panel--collapsable {
+  .Panel-parentRow { // 3
```
Comment (saralohr) [inline src/components/Panel.scss]: Could this be a modifier instead? ie `Panel-row--parent`

File: src/components/Panel.scss [hunk 2]
```diff
@@ -240,3 +241,69 @@ $Panel-header-text-color: $cu-info !default;
...
+
+.Panel-childRows {
+  background: $cu-middleground;
+  height: 0;
+  overflow: hidden;
+  &:not(:last-child) {
+    border-top: 1px solid $cu-divider--light;
+  }
+  .Panel-row {
+    border: none ;
```
Comment (saralohr) [inline src/components/Panel.scss]: Nit: extra space before semicolon.

File: src/components/Panel.scss (hunk 1 above)
Comment (williamdingwall) [reply src/components/Panel.scss]: yep good call

File: src/components/Panel.scss (hunk 2 above)
Comment (williamdingwall) [reply src/components/Panel.scss]: updated

**Pull request merged**